global/lang/en_GB/navbar.json

```
@@ -18,10 +18,18 @@
     "settings.global.language": "language",
     "settings.profiles.title": "profiles",
     "settings.profiles.showMedalsFromAllModes": "show medals from all modes",
+    "settings.medals.title": "medals",
+    "settings.medals.hideMedalsWhenFilterEnabled": "completely hide medals when unobtained filter enabled",
     "notifications.title": "<strong>notifications</strong>",
+    "notifications.subtitle": "catch up with what's goin' on",
+    "notifications.count": "<strong selector=\"NotificationCount\">$1</strong> notifications",
+    "notifications.clearAll": "clear all",
     "apps.support": "support osekai!",
     "apps.twitter": "check out osekai on twitter!",
-    "apps.discord": "join the osu! Medal Hunters Discord!",
+    "apps.discord": "join the osu! Medal Hunters discord server!",

```

I'd prefer to keep Discord uppercase
```suggestion
    "apps.discord": "join the osu! Medal Hunters Discord server!",
```